dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/src/Audacia.CodeAnalysis.Analyzers/Audacia.CodeAnalysis.Analyzers.csproj

```diff
@@ -17,7 +17,8 @@
         <PackageTags>Audacia.CodeAnalysis.Analyzers, analyzers</PackageTags>
         <NoPackageAnalysis>true</NoPackageAnalysis>
         <PackageIcon>package-icon.png</PackageIcon>
-        <Version>1.9.0</Version>
+        <Version>1.9.1</Version>
```

Comment:
this is a feature so should go up to `1.10.0` :) we save the patch increments for bugfixes